packages/aws_common/test/io/aws_file_html_test.dart

```diff
@@ -53,15 +60,17 @@ void main() {
       test('should return ChunkedStreamReader over a file path', () async {
         final awsFile = AWSFile.fromPath(testFilePath);
 
+        expect(await awsFile.contentType, 'text/plain');
```

Comment:
Probably need another test where the content-type is not embedded in the URL - can add a TODO for now since it would involve spinning up an HTTP server.